src/esp/gfx/CMakeLists.txt

```diff
@@ -76,6 +78,19 @@ target_include_directories(gfx
     "${DEPS_DIR}/glad/include"
 )
 
+if(BUILD_WITH_CUDA)
+  find_package(CUDA 8.0)
+  if(CUDA_FOUND)
+    target_include_directories(gfx PUBLIC
+      "${CUDA_INCLUDE_DIRS}"
+      ${CMAKE_CURRENT_LIST_DIR}/cuda_helpers)
```

Comment:
If I see correctly, the cuda headers are needed just in cpp files, so this could be `PRIVATE` instead :)